### PR TITLE
change display for balance

### PR DESCRIPTION
**Pull request by icerove**
- delete milli token

- if amount is less than 0.00001 near token, show <0.00001 

- hover the amount to see absolute amount

- change style to style-components

### CODE REVIEW

**Comment by kcole16** [comment]
One other comment (technically out of the scope of this issue), can we align the "Send Money" button with "Balance"?

[Designs](https://projects.invisionapp.com/share/9ES2D82U8GB#/screens)

**Comment by icerove** [comment]
change it without showing absolute atto token

File: src/components/common/Balance.js [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-    let style = {
-        width: "1em",
-        height: "1em",
-        marginLeft: "3px",
-        verticalAlign: "middle"
-    }
-    let zerosSmall = "0".repeat(NOMINATION - 3)
-    return (<div>{"0." + (zerosSmall.substring(amount.length) + amount).slice(0, 5)}<img style={style} src={milli} alt="" /></div>)
+    let amountShow = convertToShow(amount)
+    console.log(amount)
```
Comment (vgrichina) [inline src/components/common/Balance.js]: looks like leftover from debugging, please remove

File: src/components/common/Balance.js [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
+    return (
+        <CustomDiv>
+            {amountShow} Ⓝ
+        </CustomDiv>
+    )
 }
 
 const convertToShow = (amount) => {
-    return (<div>{formatNEAR(amount)} Ⓝ</div>)
```
Comment (vgrichina) [inline src/components/common/Balance.js]: nit: just `return formatNEAR(amount)` is enough

File: src/components/common/Balance.js [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+    let n = formatNEAR(amount)
+    return n
 }
 
 export const formatNEAR = (amount) => {
-    if (amount.length <= NOMINATION) {
+    if (amount.length < NOMINATION - 5) {
+        return "<0.00001"
+    }
```
Comment (vgrichina) [inline src/components/common/Balance.js]: nit: `} else if {`

**Pull request merged**